.github/workflows/plot-tests.yml

```diff
@@ -1,31 +1,41 @@
-  - name: Set up Python 3.11
-    uses: actions/setup-python@v5
-    with:
-      python-version: '3.11'
+name: Mathics3 (ubuntu)
```

Comment:
Is this the name that shows up on the front page of the PR where it shows the tests that are running? If so it might be nice to have a distinctive name so one can confirm at a glance that the plot tests have finished (successfully, or not)